bin/runner.js

```diff
@@ -2,25 +2,21 @@
  * Module dependencies
  */
 const Nightwatch = require('../lib/index.js');
-const {Logger, shouldReplaceStack, alwaysDisplayError} = require('../lib/utils');
-const path = require('path');
+const {
+  Logger,
+  shouldReplaceStack,
+  alwaysDisplayError,
+  loadTSNode,
+  findTSConfigFile
+} = require('../lib/utils');
 
 try {
-  const projectTsFile = path.join(process.cwd(), 'nightwatch', 'tsconfig.json');
-  let projectTsExists = false;
-  try {
-    require(projectTsFile);
-    projectTsExists = true;
-
-    require('ts-node').register({
-      esm: false,
-      project: projectTsFile
-    });
-  } catch (err) {
-    // eslint-disable-line
-  }
-
-  Nightwatch.cli(function(argv) {
+  Nightwatch.cli(function (argv) {
+    const projectTsFilePath = findTSConfigFile();
+    if (projectTsFilePath !== '') {
+      loadTSNode(projectTsFilePath);
+    }
```

Comment:
```suggestion
    const isTypescriptProject = Utils.fileExistsSync(path.join(process.cwd(), 'tsconfig.json');
    if (isTypescriptProject) {
      const projectTsFilePath = findTSConfigFile();
      if (projectTsFilePath !== '') {
        loadTSNode(projectTsFilePath);
      } else {
        console.log('You can now run TS tests directly using Nightwatch.\nFollow this doc to learn more.');
      }
   }
```

^ This will also help in cases where a user has mistakenly deleted the `nightwatch/tsconfig.json` file and is now trying to run ts test file but is unable to do so (because we are not loading `ts-node` anymore).